plugins/email_service/app/views/email_service/emails/index.html.haml

```diff
@@ -42,4 +42,4 @@
           recipients. 
           You can also send 
           %b
-            = link_to "templated eMails", "https://documentation.global.cloud.sap/services/email-howto-templates#send-template-email", :target => "_blank"
```

Comment:
Please still use the `:target => "_blank"`. We agreed that docu links should be open in a new tab.

Reply:
# indicates the section of the html page. Github shows this (#) as comment. However It still opens in new tab.